plutus-use-cases/src/Language/Plutus/Tutorial.lhs

```diff
@@ -0,0 +1,159 @@
+plutus-use-cases: PlutusTx tutorial
```

Comment:
The syntax of this file seems funny. The text seems to use Markdown, but you're not using `markdown-unlit`, so you're not going to get valid Markdown out from the literate Haskell processor.

The other way around will be fine (evidently), the normal lhs processor will succeed in getting the code out.

Reply:
Yeah I had some trouble getting it to work properly in ghci, so I'm writing this in an lhs file (for typechecking) and when it's done I will turn it into a markdown file with the correct formatting.

Reply:
Why not do what I did for the PlutusTx tutorial? That way it gets typechecked for you when you build (admittedly it's not a great dev experience).

Reply:
In terms of how to actually do it: my one gets typechecked when the doctests run, but you can probably also make it an internal library or testsuite just to get it built.

Reply:
> Why not do what I did for the PlutusTx tutorial?

Yeah that's what I meant by "markdown", it's just a bit tedious to set it up because of symlinks on different file systems. So I decided to put it off until the content is completed